### ISSUE TITLE
Create project-level view of code list in annotator

### ISSUE THREAD

**Issue by miahass** [edgeryders/annotator_store-gem#157]
In drop-down where you see codes by "any creator" in the codes tab of annotator, create option to see list of codes by project (e.g. POPREBEL, NGI).

**Comment by albertocottica** [comment]
As I recall it, there is no information about projects in the database. After much discussion, we agreed not to have a database entity for projects (we used to call it "collection", and we imagined it as a collection of annotations). Instead, projects are identified by Discourse tags `ethno_PROJECTNAME`, which are associated to topics, not to annotations or codes.

In my own batch analysis, I solve the problem by 

1. accessing *all* annotations via API endpoint
2. filtering them by the condition "include annotations whose `topic_id` is associated to the tag `ethno-PROJECTNAME`". 
3. filtering codes by the condition "include codes whose `id` is the same as one of the `code_id`s in the annotations selected in 2. 

There is a Python function that does 1 and 2 in [the library](https://github.com/albertocottica/discourse-social-networks/blob/master/code/python%20scripts/z_discourse_API_functions.py). It is the one called `fetch_annos(tag='')`.
